### PR TITLE
Adjust timeline

### PR DESCRIPTION
**Pull request by pchampin**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star-wg-charter/pull/50.html" title="Last updated on Jun 19, 2022, 10:05 AM UTC (5d327bd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-star-wg-charter/50/0065761...5d327bd.html" title="Last updated on Jun 19, 2022, 10:05 AM UTC (5d327bd)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -427,13 +427,13 @@ <h3>
...
-            <ul class="todo">
-              <li>Month YYYY: First teleconference</li>
-              <li>Month YYYY: Requirements and Use Cases for FooML</li>
-              <li>Month YYYY: FPWD for FooML</li>
-              <li>Month YYYY: Requirements and Use Cases for BarML</li>
-              <li>Month YYYY: FPWD FooML Primer</li>
+            <ul>
+              <li>September 2022: First teleconference</li>
+              <li>Q1 2023: FPWD of all delivarables</li>
+              <li>Q4 2023: Candiate Recommendation of RDF deliverables</li>
```
Comment (hartig) [inline index.html:433]:
```suggestion
              <li>Q1 2023: FPWD of all deliverables</li>
              <li>Q4 2023: Candidate Recommendation of RDF deliverables</li>
```

**Review by hartig — APPROVED** [review]
Just a fix for two small typos.

File: index.html [hunk 2]
```diff
@@ -86,12 +84,14 @@ <h1 id="title">RDF-star Working Group Charter</h1>
       <div style="border: 1px solid darkred">
         <p><strong>DRAFT</strong></p>
         <p>This is a <em>draft</em> text, under development by the community. The goal is to, eventually, submit this draft charter proposal to an official W3C AC review.</p>
+        <!--
```
Comment (afs) [inline index.html:87]: Remove?

File: index.html [hunk 3]
```diff
@@ -86,12 +84,14 @@ <h1 id="title">RDF-star Working Group Charter</h1>
...
         <p>Any text rendered like <span class=issue>this</span> refers to content that must be finalized before the charter is sent to AC review.</p>
+        -->
         <p>Any text rendered like <span class=todo>this</span> refers to content that must be updated when the final charter is published at the latest (e.g., adjusting hyperlinks).</p>
       </div>
 
 
-      <p class="mission">The <strong>mission</strong> of the <a href=""><i>RDF-star</i> Working Group</a> is to <span class="issue">extend a set of RDF and SPARQL related recommendations, with the ability to concisely represent and query statements about statements.</span></p>
+      <p class="mission">The <strong>mission</strong> of the <a href=""><i>RDF-star</i> Working Group</a> is to <span>extend a set of RDF and SPARQL related recommendations, with the ability to concisely represent and query statements about statements.</span></p>
```
Comment (afs) [inline index.html:94]:
```suggestion
      <p class="mission">The <strong>mission</strong> of the <a href=""><i>RDF-star</i> Working Group</a> is to extend a set of RDF and SPARQL related recommendations, with the ability to concisely represent and query statements about statements.</p>
```

**Review by afs — APPROVED** [review]
Couple of small things noticed.

**Review by gkellogg — APPROVED** [review]

**Pull request merged**